aws/s3_test.go

```diff
@@ -60,21 +76,22 @@ func (b TestS3Bucket) create(svc *s3.S3) error {
 
 	// Add default tag for testing
 	var awsTagSet []*s3.Tag
-	awsTagSet = append(awsTagSet, &s3.Tag{Key: aws.String("cloud-nuke-test"), Value: aws.String("true")})
```

Comment:
Why remove this?

Reply:
This prevented testing the situation of deleting an untagged bucket. The earier PR was so focused on deletion through tags, that the untagged scenario went untests. Added that in this PR + for test cases we are passing "cloud-nuke-test" when testing for tagged scenarios.